properties_pane/field_level/fieldLevelConfig.json

```diff
@@ -1249,6 +1249,10 @@ making sure that you maintain a proper JSON format.
 					"level": "parent",
 					"key": "childType",
 					"value": "array"
+				}, {
```

Comment:
minor UI change for definitions, because after RE all definitions is required by default